Homework/css/styles.css

```diff
@@ -0,0 +1,95 @@
+
+body {	
+
+	background: rgb(197,222,234); /* Old browsers */
```

Comment:
@JsonLitz So this whole background section... I like the code... but would you be able to tell me what it's doing? Also, what are those -moz- / -webkit- things? This is great code, I just want to challenge you to understand what's going on here. Good find here.